dll/shellext/fontext/fontext.cpp

```diff
@@ -54,7 +54,9 @@ STDAPI DllGetClassObject(REFCLSID rclsid, REFIID riid, LPVOID *ppv)
 STDAPI DllRegisterServer()
 {
     WCHAR Path[MAX_PATH] = { 0 };
-    static const char DesktopIniContents[] = "[.ShellClassInfo]\r\nCLSID={BD84B380-8CA2-1069-AB1D-08000948F534}\r\n";
+    static const char DesktopIniContents[] = "[.ShellClassInfo]\r\n"
+        "CLSID={BD84B380-8CA2-1069-AB1D-08000948F534}\r\n"
+        "IconResource=%SystemRoot%\\system32\\shell32.dll,-39\r\n"; // IDI_SHELL_FONTS_FOLDER
```

Comment:
try the following:
```suggestion
        "IconResource=%SystemRoot%\\system32\\shell32.dll,-" _CRT_STRINGIZE(IDI_SHELL_FONTS_FOLDER) "\r\n";
```
The `_CRT_STRINGIZE` macro is automatically included from crtdefs.h included as part of the CRT.

Reply:
I can't really include shell32's private header that defines `IDI_SHELL_FONTS_FOLDER`, can I?

Reply:
ah yes, my bad (you would need to include shell's own resource.h or whatever name) 😭